### PR TITLE
testsuite: r.blend quoting test

### PR DESCRIPTION
**Pull request by neteler**
Added test script which implements the test suggested in PR #277

Please check, @marisn

### CODE REVIEW

File: scripts/r.blend/testsuite/test_r_blend_quoting.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+    @classmethod
+    def setUpClass(cls):
+        """Create maps in a small region."""
+        # create a mapset with a name is also a valid mathematical expression
+        cls.runModule("g.mapset", flags="c", mapset=cls.mapset_name)
+        cls.mapsets_to_remove.append(cls.mapset_name)
+        run_command('g.copy', raster=cls.map1+'@PERMANENT,'+cls.map1)
+        cls.runModule('g.region', raster=cls.map1, flags='p')
+        run_command('d.mon', start='wx0')
```
Comment (marisn) [inline scripts/r.blend/testsuite/test_r_blend_quoting.py]: I would leave calls to any d.* modules out.

File: scripts/r.blend/testsuite/test_r_blend_quoting.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        """Remove temporary data"""
+        gisenv = SimpleModule('g.gisenv', get='GISDBASE')
+        cls.runModule(gisenv, expecting_stdout=True)
+        gisdbase = gisenv.outputs.stdout.strip()
+        gisenv = SimpleModule('g.gisenv', get='LOCATION_NAME')
+        cls.runModule(gisenv, expecting_stdout=True)
+        location = gisenv.outputs.stdout.strip()
+        cls.runModule('g.remove', flags='f', type='raster',
+                      name=(cls.temp1, cls.temp2, cls.temp3))
+        run_command('d.mon', stop='wx0')
```
Comment (marisn) [inline scripts/r.blend/testsuite/test_r_blend_quoting.py]: Ibid.

File: scripts/r.blend/testsuite/test_r_blend_quoting.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+#r.grow input=elevation@1234-56-78 output=grown
+#Observe output: syntax error, unexpected INTEGER, expecting VARNAME or NAME
+
+        # should not lead to syntax error, unexpected INTEGER, expecting VARNAME or NAME
+        module = SimpleModule('r.blend', first=self.map2, second=self.map1+'@'+self.mapset_name,
+                              output='elev_shade_blend')
+        self.assertModule(module)
+
+        run_command('d.rgb', red=self.temp1, green=self.temp2,
```
Comment (marisn) [inline scripts/r.blend/testsuite/test_r_blend_quoting.py]: This call also does not add a lot for the actual test. Just kick it out and ship it!

**Comment by neteler** [comment]
Feel free to submit changes to this PR

**Comment by neteler** [comment]
Thanks for your efforts, works fine now.

File: scripts/r.blend/testsuite/test_r_blend_quoting.py [hunk 4]
```diff
@@ -0,0 +1,65 @@
+"""
+Created on Mon 17 Feb 2020 02:27:26 PM UTC
+
+@author: Markus Neteler, upon https://github.com/OSGeo/grass/pull/277
```
Comment (neteler) [inline scripts/r.blend/testsuite/test_r_blend_quoting.py]:
```suggestion
@author: Markus Neteler and Māris Nartišs, upon https://github.com/OSGeo/grass/pull/277
```

**Pull request merged**